src/sdk/namespace_fs.js

```diff
@@ -980,7 +1069,7 @@ class NamespaceFS {
             is_latest: true,
             delete_marker: false,
             tag_count: 0,
-            xattr: {},
+            xattr: _.isEmpty(xattr) ? {} : get_xattr_read(xattr),
```

Comment:
move this _.isEmpty check into the function
```suggestion
            xattr: get_xattr_from_fs(fs_xattr),
```